htdocs/societe/card.php

```diff
@@ -3339,3 +3359,46 @@ function init_check_no_email(input) {
 // End of page
 llxFooter();
 $db->close();
+
+?>
+
+<script>
+// Function to retrieve VAT details from the Greek Ministry of Finance GSIS SOAP web service
+function GRVAT(a, u, p, myafm) {
+  var afm = a.replace(/\D/g, ""); // Remove non-digit characters from 'a'
+
+  $.ajax({
+	type: "GET",
+	url: "/societe/checkvat/checkVatGr.php",
+	data: { u, p, myafm, afm }, // Set request parameters
```

Comment:
I think only the param afm must be oassed as parameter the the json ajax service. Now security data as retreived directly by the service and are no more exposed as parameter. Can you check with only afm and confirm my thinking ?

Reply:
Yes it works, fixed thank you.